README.md

```diff
@@ -94,6 +97,10 @@ violations for your environment.
 --                   idea in production, as it can reduce the average additional latency
 --                   caused by this module if anything goes wrong with the underlying
 --                   infrastructure. (defaults to disabled)
+--    statsd_host - Host of statsd collector. Setting this will enable statsd metrics collection
+--    statsd_port - Port of statsd collector. (defaults to 8125)
+--    statsd_buffer_size - Statsd buffer table length at which stats should be sent to
+--                         the collector. (defaults to 100)
```

Comment:
It's unclear here if this is item count or something like the buffer size in bytes